tests/pack/filelimit.c

```diff
@@ -0,0 +1,138 @@
+#include "clar_libgit2.h"
+#include "mwindow.h"
+#include "global.h"
+
+#include <git2.h>
+#include "git2/sys/commit.h"
+#include "git2/sys/mempack.h"
+
+static size_t _expected_open_mwindow_files = 0;
+static size_t _original_mwindow_file_limit = 0;
```

Comment:
Leading underscores are reserved by the C/libc implementations and thus aren't allowed to be used. Sorry to be nitpicky here, I bet you'd not even have to look far to found counterexamples in the current code already where we're doing exactly what you're doing here.